configuration/configuration.go

```diff
@@ -0,0 +1,34 @@
+package configuration
+
+import (
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/ui"
+	"os"
+	"strconv"
+)
+
+const (
+	ChunkSizeInMBEnv     = "CHUNK_SIZE_IN_MB"
+	TargetURLEnv         = "DEPLOY_SERVICE_URL"
+	DefaultChunkSizeInMB = uint64(45)
+)
+
+func GetChunkSizeInMB() uint64 {
+	chunkSizeInMb, isSet := os.LookupEnv(ChunkSizeInMBEnv)
+	if isSet {
+		parsedChunkSizeInMb, err := strconv.ParseUint(chunkSizeInMb, 10, 64)
+		if err == nil && parsedChunkSizeInMb != 0 {
+			ui.Say("Attention: You've specified a custom chunk size (%d MB) via the environment variable \"%s\".", parsedChunkSizeInMb, ChunkSizeInMBEnv)
+			return parsedChunkSizeInMb
+		}
+		ui.Warn("Attention: You've specified an INVALID custom chunk size (%s) via the environment variable \"%s\". Using default: %d", chunkSizeInMb, ChunkSizeInMBEnv, DefaultChunkSizeInMB)
+	}
+	return DefaultChunkSizeInMB
+}
+
+func GetTargetURL() string {
+	targetURL := os.Getenv(TargetURLEnv)
```

Comment:
What is the difference between the LookupEnv and the Getenv?

Reply:
LookupEnv returns a boolean flag that indicates whether the variable was set at all. In this method we don't care about that, since there's no default value.